src/EA.Weee.Web/Areas/AATFReturn/Controllers/SentOnCreateSiteOperatorController.cs

```diff
@@ -67,7 +67,7 @@ public virtual async Task<ActionResult> Index(SentOnCreateSiteOperatorViewModel
 
                     await client.SendAsync(User.GetAccessToken(), request);
 
-                    return RedirectToAction("Index", "Holding", new { organisationId = viewModel.OrganisationId });
+                    return RedirectToAction("Index", "ObligatedSentOn", new { organisationId = viewModel.OrganisationId, aatfId = viewModel.AatfId, returnId = viewModel.ReturnId, weeeSentOnId = viewModel.WeeeSentOnId });
```

Comment:
Can we change this to use the AatfRedirect class, should be able to re-use a route and pass nec3essary parameters, if not can create new.